fed/mosaic-sns/src/main/java/org/eclipse/mosaic/fed/sns/model/AdhocTransmissionModel.java

```diff
@@ -50,19 +50,36 @@ TransmissionResult simulateTransmission(RandomNumberGenerator randomNumberGenera
     }
 
     /**
-     * Method to be implemented by extensions of {@link AdhocTransmissionModel}, calculating transmissions using topocast.
+     * Method to be implemented by extensions of {@link AdhocTransmissionModel}, calculating transmissions using topocast with
+     * only one single hop, either broadcast or unicast.
      *
      * @param senderName            The sender of the transmission.
      * @param receivers             The receivers of the transmission.
      * @param transmissionParameter Data class holding the maximumTtl, the {@link Delay} and the current map of simulated entities
      * @param currentNodes          a reference to all currently online nodes
      * @return Map of the receivers and their transmission results.
      */
-    public abstract Map<String, TransmissionResult> simulateTopocast(
+    public abstract Map<String, TransmissionResult> simulateSinglehop(
```

Comment:
For clarification one could think of using `simulateTopocastSinglehop` or something where `topo` is still part of the name

Reply:
Ok, I'm with you here. Should we rename `simulateTopologicalUnicast` to `simulateTopologicalMultihopUnicast` then? It would be more clear, but also a very long name.

Reply:
I would leave it as it is now :) Because multihop here is not really exclusive, I mean: yes it CAN be multihop but it also can be reachable with one single hop, so I don't find it important enough to mention in the name.